toolbox/certbot/renew.sh

```diff
@@ -1,12 +1,14 @@
 #!/bin/bash
 
+echo ""
+echo "$(date '+%Y-%m-%d %H:%M:%S')"
 echo "Renewing the certificates for the Airflow stack"
+
+
 /usr/airflow/atd-airflow/toolbox/certbot/renew_domain_with_certbot.sh airflow.austinmobility.io
 /usr/airflow/atd-airflow/toolbox/certbot/renew_domain_with_certbot.sh airflow-workers.austinmobility.io
 
-# Build the HAProxy image with the renewed certificates
 cd /usr/airflow/atd-airflow
-docker compose build haproxy
 
 # Restart the HAProxy stack to use renewed certificates
-docker compose restart haproxy
+BUILDKIT_PROGRESS=plain docker compose restart haproxy
```

Comment:
Ask the `docker` `compose` service to print its status plainly which works better in log files.